docs/source/secrets.rst

```diff
@@ -1,33 +0,0 @@
-Secrets
```

Comment:
@ryandeivert i might be derp'ing, but `docs/source/outputs.rst` doesn't appear to cover the new way of doing secrets, where is it covered?